config/externalname.go

```diff
@@ -133,6 +133,8 @@ var TerraformPluginSDKExternalNameConfigs = map[string]config.ExternalName{
 	// authorization
 	"azurerm_resource_group_policy_assignment": config.TemplatedStringAsIdentifier("name", "{{ .parameters.resource_group_id }}/providers/Microsoft.Authorization/policyAssignments/{{ .external_name }}"),
 	"azurerm_role_assignment":                  config.IdentifierFromProvider,
+	"azurerm_pim_eligible_role_assignment":     config.IdentifierFromProvider,
+	"azurerm_pim_active_role_assignment":       config.IdentifierFromProvider,
```

Comment:
Could you add the comment lines about import?

```suggestion
        // Pim Eligible Role Assignments can be imported using the resource id
	"azurerm_pim_eligible_role_assignment":     config.IdentifierFromProvider,
	// Pim Active Role Assignments can be imported using the resource id
	"azurerm_pim_active_role_assignment":       config.IdentifierFromProvider,
```